package.json

```diff
@@ -17,6 +17,7 @@
       "style": "./dist/css/index.css"
     },
     "./client": {
+      "types": "./dist/client/index.d.ts",
```

Comment:
I was unable to import the package using ts

Reply:
Interesting it helped (as there are no types anyway :man_shrugging: )